### PR TITLE
add(epp): hyper-v nscp api staging

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Target version

- [X] 20.10.x (staging)
- [X] 21.04.x (staging)
- [X] 21.10.x (staging)
- [ ] 22.04.x (next)

### CODE REVIEW

File: versioned_docs/version-20.04/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+    --password=centreon \
+    --insecure \
+    --http-backend=curl \
+    --command=check_centreon_plugins \
+    --arg='apps::microsoft::hyperv::2012::local::plugin' \
+    --arg='node-vm-status' \
+    --arg='--help'
+```
+
+All available options for a given mode can be displayed by adding the
```
Comment (lchrdn) [inline versioned_docs/version-20.04/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md:264]:
```suggestion
All available modes can be displayed by adding the 
```

File: versioned_docs/version-20.10/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    --password=centreon \
+    --insecure \
+    --http-backend=curl \
+    --command=check_centreon_plugins \
+    --arg='apps::microsoft::hyperv::2012::local::plugin' \
+    --arg='node-vm-status' \
+    --arg='--help'
+```
+
+All available options for a given mode can be displayed by adding the
```
Comment (lchrdn) [inline versioned_docs/version-20.10/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md:264]:
```suggestion
All available modes can be displayed by adding the 
```

File: versioned_docs/version-21.04/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    --password=centreon \
+    --insecure \
+    --http-backend=curl \
+    --command=check_centreon_plugins \
+    --arg='apps::microsoft::hyperv::2012::local::plugin' \
+    --arg='node-vm-status' \
+    --arg='--help'
+```
+
+All available options for a given mode can be displayed by adding the
```
Comment (lchrdn) [inline versioned_docs/version-21.04/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md:264]:
```suggestion
All available modes can be displayed by adding the 
```

File: versioned_docs/version-21.10/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+    --password=centreon \
+    --insecure \
+    --http-backend=curl \
+    --command=check_centreon_plugins \
+    --arg='apps::microsoft::hyperv::2012::local::plugin' \
+    --arg='node-vm-status' \
+    --arg='--help'
+```
+
+All available options for a given mode can be displayed by adding the
```
Comment (lchrdn) [inline versioned_docs/version-21.10/integrations/plugin-packs/procedures/virtualization-hyperv-nscp-restapi.md:264]:
```suggestion
All available modes can be displayed by adding the 
```

**Comment by lchrdn** [comment]
I think you have to add your new documentation name in the sidebars for each version (fr and en), for example here https://github.com/centreon/centreon-documentation/blob/staging/versioned_sidebars/version-21.10-sidebars.json#L3296

**Review by Sims24 — CHANGES_REQUESTED** [review]
Missing sidebar update

**Review by lchrdn — APPROVED** [review]

**Comment by Sims24** [comment]
Changes will be applied in another PR

**Pull request closed without merging**